PhysicsTools/PatAlgos/python/slimming/modifiedElectrons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@slava77 - as you suggest - a "dangling module definition" would run if (only if) the output module or some other producer/filter asks for it. So, doing wildcards in the output module is the one potential pitfall.